pkg/controllers/networkpolicy/sync.go

```diff
@@ -162,6 +162,7 @@ func (a *appliedToSecurityGroup) sync(syncContent *securitygroup.Synchronization
 	}
 
 	if indexerUpdate {
+		_ = a.updateAddrGroupReference(r)
```

Comment:
How is the old address group reference added to AT?

Reply:
The old rule referencing previous address group is synced into the indexer for the current AT, and have an indexer update, so this func call will update the reference in AT.

Reply:
Done